telemetry/src/Control.js

```diff
@@ -544,6 +544,24 @@ class Control extends Component {
                   </Grid>
                 </Grid>
               </Grid>
+              <Grid item={1} xs={3} className={classes.item}>
+                <Grid container spacing={1} direction='column'>
+                  <Grid item>
+                    <ButtonGroup
+                      open={comms.activateLoxTankMidHtr}
+                      close={comms.deactivateLoxTankMidHtr}
+                      text='Main Valve Vent'
+                    />
+                  </Grid>
+                  <Grid item>
+                    <ButtonGroup
+                      open={comms.activateLoxTankBottomHtr}
+                      close={comms.deactivateLoxTankBottomHtr}
+                      text='RQD'
+                    />
+                  </Grid>
+                </Grid>
+              </Grid>
```

Comment:
Gotta update the naming

Reply:
Yea, aware of that - making a quick n dirty fix while at RFS. Wanted to still get what we used merged into main, and will make a new branch with updated backend